### ISSUE TITLE
New Features

### ISSUE THREAD

**Issue by ktquez** [ktquez/vue-head#23]
[ ] Support SSR

**Comment by gaeldestrem** [comment]
Hi :)

I'm really interested by the SSR support, are you planning to release this feature soon?

Thank you :)

**Comment by ktquez** [comment]
@gaeldestrem 
Yes, I'll be testing this week how to get and will create an example.

**Comment by kasperhelweg** [comment]
How's it going with the SSR support? I would love to help out if you need it. Cheers!

**Comment by Leeaandrob** [comment]
Hello @ktquez vue-head support ssr?

**Comment by ktquez** [comment]
@Leeaandrob 
I'm sad that I do not have much time to do a refactory and add SSR support. But it's something I want to do, but I have no foresight.

There is an option that is the [vue-meta](https://github.com/declandewet/vue-meta)

**Comment by Leeaandrob** [comment]
Ok thanks @ktquez
